### ISSUE TITLE
Tests depending on Atomistica are skipped

### ISSUE THREAD

**Issue by jameskermode** [libAtoms/matscipy#4]
It would be good to install Atomistica in the Travis-CI environment  @pastewka  do you think this is possible?

**Comment by pastewka** [comment]
Yes should be possible. I wanted to upload Atomistica to pypi anyway, maybe this would be the easiest way. You could then install with pip.

**Comment by pastewka** [comment]
Atomistica in now on pip but does not play well with OS X install
